src/prebid.js

```diff
@@ -41,6 +41,7 @@ $$PREBID_GLOBAL$$._bidsReceived = [];
 $$PREBID_GLOBAL$$._winningBids = [];
 $$PREBID_GLOBAL$$._adsReceived = [];
 $$PREBID_GLOBAL$$._sendAllBids = false;
+$$PREBID_GLOBAL$$._bidderSequence = null;
```

Comment:
We shouldn't make values global unless required. Can you make this a private variable of `adaptermanager` ?